### ISSUE TITLE
I am unable to submit questions to "IBM Developer Answers"

### ISSUE THREAD

**Issue by TerryBoyd** [IBM/developer#44]
I have tried to submit the same question many times this morning.
Each time, I observe an error indicating...

Unable to execute your request
We're sorry, but we're unable to execute your request. You are probably seeing this error because your session expired due to inactivity. When using IBM Developer Answers, constructing your posts offline and then posting them in a timely fashion once on the site should prevent inadvertent loss of data. Please attempt to visit the homepage of this site and then try your action again. If you believe this error is incorrect, please contact the site administrator(s).

There is NO way I am taking too long to submit my request.
I expect the site believes I am attempting to submit attack code, since the URL of this page is...

https://developer.ibm.com/answers/static/csrfAttackDetected.html

This is an URGENT issue - I need to submit this question!

**Comment by LindaMeyer** [comment]
@TerryBoyd  Sorry to hear that you are having a problem with IBM Developer Answers. 

Try logging out and back in. If that doesn't work, try using a private browser session and then logging in, or clearing ibm.com cookies and then logging in.

**Comment by TerryBoyd** [comment]
Hi Linda,

I have tried all suggested methods, and continue to observe the error upon submission of a question.

I suspect it may be because IBM has TWO separate logins for my account (tboyd@saxonsystems.com.au).
My original account used only the short version of my id (i.e.  'tboyd').
I noticed that they are BOTH valid accounts, but have a different 'screen name' (one has no space between my first name and surname).
I really need to post some questions soon.Have you any other suggestions?

Cheers!

**Comment by LindaMeyer** [comment]
Hi Terry, I will reach out to our support team. It isn't standard business hours here in the US right now, so please be patient, but I will try to get you an answer asap.  Your response in this Github issue (which is available publicly) includes your phone number and address. You may want to edit that to remove it.

**Comment by jantley-ibm** [comment]
Hi Terry, which profile do you want to keep and which profile do you want me to deactivate?

**Comment by TerryBoyd** [comment]
Please remove the 'tboyd' account - I'll retain the one with my email address.

Thanks!

**Comment by jantley-ibm** [comment]
Having some trouble, please bear with me while I continue to try to deactivate the account

**Comment by jantley-ibm** [comment]
Sorry, these functions are currently nonresponsive.  We'll need server admin support to try to rectify the situation.  Could I post your question for you in the meantime?

**Comment by TerryBoyd** [comment]
Yes... please do!

Title:

Notes C&S is broken depending on method to create meeting in different time zone

Body:

We have several users here in Australia that create meetings in different time zones (e.g. NZ).

Depending on HOW they create those meetings, the Notes client behaves very differently...

If they open their calendar, click the 'New' action button to create a meeting, and reset all the date/time values the issue does not appear
If they simply double-click on the required time slot, issue appears consistently.

An example... 

A meeting is created with a start time (1:00PM) and an end time (2:00PM) by double clicking the relevant calendar time slot
The time zone is then switched from 'Local' to 'GMT+12' (Auckland), and the local times are then displayed on the right hand side (11:00AM - 12:00PM)
When the 'Save & Send invitations' action button is clicked, however, the meeting times are converted to 3:00PM - 4:00PM (in the NZ time zone) with the local meeting time updated to 1:00PM - 2:00PM!

It seems the time zone switcher may be causing the issue, as I performed a subsequent test...

A meeting is created with a start time (1:00PM) and an end time (2:00PM) by double clicking the relevant calendar time slot
The time zone is then switched from 'Local' to 'GMT+12' (Auckland), and the local times are then displayed on the right hand side (11:00AM - 12:00PM)
The meeting times are then changed to something else and then RE-SELECTED to make them the original 1:00PM and 2:00PM values
When the 'Save & Send invitations' action button is clicked, the meeting times remain exactly as they should be

My original question was asked in the Notes/Domino 9.0 Social Edition Forum.
The answer boiled down to removal of the following entries from the local NOTES.INI file:

Timezone=-10
DSTLAW=10,1,1,4,1,1
DST=1

However, those entries are added BACK into the NOTES.INI file each time the Notes client is re-started.
There are certainly no Policy settings to force the creation of those entries.
So my questions are...

Are those NOTES.INI entries mandatory now, and added automatically by the Notes client?
If so, can we prevent the automatic creation of those entries somehow?
If we can't prevent the creation of those entries, then how can we resolve this C&S issue?


Thanks so much for any/all feedback!

**Comment by jantley-ibm** [comment]
Encountering the same csrf issue.  Trying to get a server admin's attention.

**Comment by jantley-ibm** [comment]
Was able to get your question posted and de-activate the account you specified.  Sorry for the trouble.

**Comment by ErwanLeBras** [comment]
I get the same error each time I try to ask a new question

**Comment by jantley-ibm** [comment]
Erwan, please provide a link to your profile page on the Answers site and I will look into it.

**Comment by ErwanLeBras** [comment]
Hi @jantley-ibm 
Thanks !
My profile is 
https://developer.ibm.com/answers/users/250283/ilog-elb/

Today I was able to post my question (and I ended up with 3 posts, same question same hour so did delete two of them).
 
What 's different between today and the days before ?
Yesterday I was using VPN
Today I am on IBM Site. 
Don't know if this is the reason or not but worth checking. 

Thanks !

Erwan
